### ISSUE TITLE
WSL - reinstall not working

### ISSUE THREAD

**Issue by cvkmohan** [elixir-tools/elixir-tools.vscode#98]
![image](https://github.com/user-attachments/assets/819add7c-bd79-4456-b0e2-8593717d3a30)
I uninstalled and reinstalled the extension. However, it shows executable not found. How do I force it to download it again? Any number of reinstalls is not helping.

**Comment by mhanberg** [comment]
Have you changed the installation directory of next LS in the settings?

**Comment by cvkmohan** [comment]
No. No change. In fact I found the nextls executable in `~/.cache/elixir-tools/nextls/bin` - I deleted it and reinstalled - no use. Similarly, I found `~/.vscode-server/elixir-tools` where the vscode installation. I tried removing that folder and reinstalling. In both cases the executable is neither downloaded nor could be located.

**Comment by mhanberg** [comment]
can you show what the permissions are on that directory? the cache dir one

**Comment by cvkmohan** [comment]
![image](https://github.com/user-attachments/assets/3b1e395a-3324-4947-b801-b4be27326354)

**Comment by mhanberg** [comment]
sorry, i meant on `~/.cache/elixir-tools` and also the sub folders, I can't tell what they are from the screenshot

**Comment by cvkmohan** [comment]
![image](https://github.com/user-attachments/assets/dc45286c-83b7-4f05-bce7-b546c9cb9479)
All folders have the same permission. So, I thought names will not be necessary. My bad.

**Comment by mhanberg** [comment]
thanks, have you tried deleting ~/.cache/elixir-tools or have you just deleted the actual nextls binary?

**Comment by cvkmohan** [comment]
I deleted the entire folder. It is not getting created. So, I created - and - then downloaded the binary and kept in place. Still, vscode extension is not recognizing this.

**Comment by mhanberg** [comment]
Hmm, very odd. 

I think based on the logs, it is basically hanging when it tries to make that directory.

Just covering my bases, have you completely quit vscode and reopened, and potentially restart your computer?

**Comment by cvkmohan** [comment]
Yes. Though it is WSL - belongs to Windows family. So, yes - tried that. In the screenshot - the force nextls download flag is false. Is there any way I can set it to true? Or not needed?

**Comment by mhanberg** [comment]
That is just for when the builtin auto updater inside nextls is buggy and i need a way to tell the extension to reinstall it.
